src/styles/product-card/card.scss

```diff
@@ -0,0 +1,126 @@
+.main {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(4, 198px);
+  gap: 46px 50px;
+  width: 944px;
+  height: 862px;
```

Comment:
Let's remove this because on smaller widths it looks weird and with horizontal scrollbar 
```suggestion
```

![image](https://github.com/mate-academy/layout_catalog/assets/46566190/4db74baf-fa32-4615-8bd6-32e845ab81ed)